src/ir/properties.h

```diff
@@ -85,6 +85,32 @@ inline bool isConstantExpression(const Expression* curr) {
   return false;
 }
 
+inline Literal getSingleConstantValue(const Expression* curr) {
```

Comment:
It seems like the natural name for these would be `getSingleLiteral / getLiterals`?

If you prefer to rename `Literal` entirely, I'm open to that. Maybe `Constant`? Doesn't need to block this PR, but let's decide on this soon and not keep making such a future renaming harder and harder.

Or, if we don't want to do such an eventual renaming, then `getSingleLiteral / getLiterals` seems to make more sense unless I'm missing something.

Reply:
No, you're right, `getSingleLiteral` and `getLiterals` make more sense. This `ConstantValue` naming came from trying to match the naming scheme of `isConstantExpression`, not from a place of trying to subvert the standard `Literal` name.